### DISCUSSION TITLE
QUESTION: how could I know the version of the github runner?

### DISCUSSION THREAD

**Discussion by shenmou98** [actions/actions-runner-controller#619] [Questions] [answered]
QUESTION: how could I know the version of the github runner is running?

**Comment by mumoshu** [comment, marked as answer]
@shenmou98 Hey! The only way would be to run `kubectl get po -o yaml` and see the image tag of the runner image. Also, please consider using GitHub Actions for questions and discussions. Issues are for concrete bug reports and feature requests :)

**Reply by toast-gear** [reply]
This may not be enough too if you are running a static runner since the feature was introduced. Runners self-update atm and this can't be opted out of. This isn't a problem when your runners are ephemeral (as they are by default with this project) as they don't get a chance to update, however we've had static runner support in HEAD for a while (https://github.com/actions-runner-controller/actions-runner-controller/pull/465) and is now in the latest release 💪 so it is something to be aware of if you are running this configuration for any runners.

The current runner version is always spat out in the `Set up job` step for all workflow runs.

**Comment by mumoshu** [comment]
Moving to Discussions

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
